web/client/epics/__tests__/annotations-test.js

```diff
@@ -1043,6 +1043,45 @@ describe('annotations Epics', () => {
         store.dispatch(action);
 
     });
+    it('clicked on map selecting a Circle ', (done) => {
+        store = mockStore( {
+            ...defaultState,
+            annotations: {
+                ...defaultState.annotations,
+                config: {
+                    geodesic: true
+                },
+                editing: {
+                    ...defaultState.annotations.editing,
+                    features: [{
+                        type: "Feature",
+                        geometry: {
+                            type: "Polygon",
+                            coordinates: [[[1, 2], [1, 3], [1, undefined], [1, 5], [1, 2]]]
```

Comment:
let's use number in mocked coords
```suggestion
                            coordinates: [[[1, 2], [1, 3], [1, 1], [1, 5], [1, 2]]]
```